demo/src/screens/SettingsScreen.js

```diff
@@ -5,19 +5,21 @@ import {StyleSheet, I18nManager} from 'react-native';
 import {Colors, View, Text, Picker, Toast, Switch} from 'react-native-ui-lib'; //eslint-disable-line
 import {navigationData} from './MenuStructure';
 
-
 const none = {label: '[None]', value: ''};
 const playgroundScreen = {label: 'Playground', value: 'unicorn.PlaygroundScreen'};
 
 class SettingsScreen extends Component {
+
   constructor(props) {
     super(props);
 
     const data = props.navigationData || navigationData;
     const playground = props.playground || playgroundScreen;
-    
+    const extraSettingsUI = props.extraSettingsUI;
+
     this.state = {
       showRefreshMessage: false,
+      extraSettingsUI,
```

Comment:
Why are you keeping `extraSettingsUI` in the component state? Why not use it directly from prop in your render function ?

Reply:
fixed